xain_fl/grpc/coordinator.py

```diff
@@ -656,26 +743,36 @@ def monitor_heartbeats(
         next_expiration = coordinator.participants.next_expiration() - time.time()
 
         logger.debug("Monitoring heartbeats in %.2f", next_expiration)
+
+        influxdb_metrics.write_number_of_participants(
+            influxdb_client, coordinator.participants.len()
+        )
+
         time.sleep(next_expiration)
 
 
-def serve() -> None:
+def serve(coordinator: Coordinator, host="[::]", port=50051) -> None:
     """Main method to start the gRPC service.
 
     This methods just creates the :class:`~.Coordinator`, sets up all threading
     events and threads and configures and starts the gRPC service.
     """
-    coordinator = Coordinator()
     terminate_event = threading.Event()
     monitor_thread = threading.Thread(
         target=monitor_heartbeats, args=(coordinator, terminate_event)
     )
 
-    server = grpc.server(futures.ThreadPoolExecutor(max_workers=10))
+    options = [
```

Comment:
These options remove the size limit of sent and received messages. Right now the reason is for the messages that send numpy arrays. In the future the numpy arrays may not be shared through gRPC or if they are we need to choose and bound the maximum message size.

Reply:
Will add a task to keep this on the radar.